### ISSUE TITLE
[Questions] With the og archive site down, what can be done about it?

### ISSUE THREAD

**Issue by omikamidraws** [arichornlover/uYouEnhanced#767]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

✅ Yes, I believe this is a bug. I will open a new issue with the bug template

### My question

I'm not asking about what you and the team are going to do. I already know you all are trying your best to resolve from the **Issues** tab I've read on this project.  I want to ask if there is anything we, the audience who appreciate the modifications, can do anything to help.

### Additional context

Basically yesterday, I was able to install v19.40.4-3.0.4 when there was a opportunity gap to install it from the source, evident in issue #766 . Funny enough, I didn't know about the problem that time and just got lucky.

Next day, tried to install it into my relative's phone, but it didn't work as "the file doesn't exist". Checked everything to find out it was the v19.33.2-3.0.4 and I stumbled across issue #766 specifically after some trial and error.
